topics/proteomics/tutorials/neoantigen-2-non-normal-database-generation/tutorial.md

```diff
@@ -0,0 +1,666 @@
+> 2. What information is typically included in the title and sequence columns in a FASTA file?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Converting tabular data back to FASTA format makes it compatible with a wide range of bioinformatics tools that require FASTA input. It allows the sequence data, now standardized and labeled, to be used in additional analyses, such as protein structure prediction or functional annotation.
+> > 2. The title column usually contains a unique identifier or label for each sequence, often including relevant annotations. The sequence column contains the actual DNA, RNA, or protein sequence. Together, these provide essential details for recognizing and analyzing each sequence entry in downstream applications.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Merging Single amino acid variant databases with FASTA Merge Files and Filter Unique Sequences
+FASTA Merge Files and Filter Unique Sequences is a tool that combines multiple FASTA files into a single file and removes any duplicate sequences, keeping unique entries. In this task, the tool takes the FASTA file generated from the Tabular-to-FASTA step and merges it with any other FASTA files in the input list. The tool then filters the sequences to ensure that unique sequences are retained in the final output, which is important for reducing redundancy in the dataset.
+
+In this workflow, FASTA Merge Files and Filter Unique Sequences consolidate all sequence data into a single, non-redundant FASTA file. This step is essential for removing duplicate sequences, which helps streamline the dataset for further analysis. A unique sequence file reduces computational load and minimizes potential biases in downstream applications that could be affected by redundant data. We are merging the indel, snv, and rpkm databases in this step.
+
+
+> <hands-on-title> FASTA Merge Files and Filter Unique Sequences </hands-on-title>
+>
+> 1. {% tool [FASTA Merge Files and Filter Unique Sequences](toolshed.g2.bx.psu.edu/repos/galaxyp/fasta_merge_files_and_filter_unique_sequences/fasta_merge_files_and_filter_unique_sequences/1.2.0) %} with the following parameters:
+>    - *"Run in batch mode?"*: `Merge individual FASTAs (output collection if the input is a collection)`
+>        - In *"Input FASTA File(s)"*:
+>            - {% icon param-repeat %} *"Insert Input FASTA File(s)"*
+>                - {% icon param-file %} *"FASTA File"*: `SNV` (output of **Tabular-to-FASTA** {% icon tool %})
+>        - In *"Input FASTA File(s)"*:
+>            - {% icon param-repeat %} *"Insert Input FASTA File(s)"*
+>                - {% icon param-file %} *"FASTA File"*: `INDEL` (output of **Tabular-to-FASTA** {% icon tool %})
+>        - In *"Input FASTA File(s)"*:
+>            - {% icon param-repeat %} *"Insert Input FASTA File(s)"*
+>                - {% icon param-file %} *"FASTA File"*: `RPKM` (output of **Tabular-to-FASTA** {% icon tool %})>
```

Comment:
this seems to imply that we renamed the output of Tabular-To-Fasta as RPKM/SNV/Indel etc, but that step is missing there, so please add that rename step to the Tabular-to-Fasta boxes above